examples/integrations/hf_trainer_api/README.md

```diff
@@ -0,0 +1,58 @@
+# Vision Transformer with HuggingFace Trainer API and Determined
+This example is adapted from the 
+[VisionTransformer example in the Hugging Face examples](https://github.com/huggingface/transformers/tree/main/examples/pytorch/image-classification). 
+It is intended to demonstrate how to use Determined callback with Hugging Face Trainer API to
+enable Determined's distributed training, fault tolerance, checkpointing and metrics reporting.
+
+## Files
+* **image_classification.py**: The code from Hugging Face that (1) loads Vision Transformer from Model Hub; (2)
+configure Trainer; (3) uses Determined callback.
+
+The key portion of the code is providing Determined callback to the Trainer in line 413:
```

Comment:
The key portion of the code passing the Determined callback to the Transformers Trainer is in line 413